frontend/packages/ceph-storage-plugin/integration-tests-cypress/views/store.ts

```diff
@@ -7,16 +7,23 @@ export enum Providers {
 
 export const testName = 'test-bucket';
 
+export enum StoreType {
+  BackingStore = 'backingstore',
+  NamespaceStore = 'namespacestore',
+}
```

Comment:
Instead of these you can use `kind` property defined in `model` of each of these CR.

Reply:
Add a comment if its also used for data-test, that the values are based on data-test fields.